packages/client/webapp/src/components/data-explorer/search-bar.vue

```diff
@@ -153,31 +149,30 @@ export default defineComponent({
 	cursor: pointer;
 }
 
-.clear-button {
-	color: white;
+button {
```

Comment:
We have a `<Button>` component

Reply:
this is a minor refactoring that i would leave for later